techniques/system/distributePolicy/1.0/rudder-rsyslog-root.st

```diff
@@ -70,3 +70,17 @@ $template RudderReportsFormat,"insert into RudderSysEvents (executionDate, nodeI
 
 ## Syslog messages from "rudder" forwarded from AIX
 :msg, ereregex, "from .*: rudder:" ~
+
+&if(SKIPIDENTIFY)&
+
+$AllowedSender UDP, &ALLOWEDNETWORK&
+$AllowedSender TCP, &ALLOWEDNETWORK&
+
+&else&
+
+&MANAGED_NODES_IP : {ip |
+$AllowedSender UDP, &ip&
```

Comment:
It's one line per IP, hence the syntax "&MANAGED_NODES_IP : {ip |" which breaks an array into components.